.ycm_extra_conf.py

```diff
@@ -5,13 +5,13 @@
 
 CONDA_PREFIX = os.environ["CONDA_PREFIX"]
 NVJITLINK_DIR = Path(__file__).parent
-VER = sys.version_info
-PYTHON_INCLUDE_NAME = "python3.10"  # Not ideal
+PYTHON_VERSION = sys.version_info
+PYTHON_INCLUDE_NAME = "python%d.%d" % PYTHON_VERSION
 
 CONDA_INCLUDE_DIR = Path(CONDA_PREFIX, "include")
```

Comment:
Maybe we could generalize to something like this?

```suggestion
BASE_INCLUDE_DIR = sysconfig.get_config_var("INCLUDEDIR")
```

Would yield other changes around this

Reply:
I think we could make this change, but I'm not sure it would yield any other changes.